package.nls.json

```diff
@@ -2,6 +2,10 @@
 "extension.runQuery":"Execute Query",
 "extension.runCurrentStatement":"Execute Current Statement",
 "extension.cancelQuery":"Cancel Query",
+"extension.addObjectExplorer":"Add Object Explorer",
+"extension.removeObjectExplorerNode":"Remove",

```

Comment:
Wouldn't something like "Show Object Explorer" be a better title? Or am I misunderstanding what this command is doing?

Reply:
Well, there's a `View: Object Explorer` already that we get for free since I added a new `Viewlet`/`View` for Object Explorer. This command basically adds a node to the tree for a new server.

Reply:
Then shouldn't it be Add Server or Add Connection or something? Right now it's implying that it's adding "Object Explorer" (whatever that means) so it seems like this could be better clarified

---
In reply to: [303570661](https://github.com/microsoft/vscode-mssql/pull/1259#discussion_r303570661) [](ancestors = 303570661)

Reply:
@Charles-Gagnon Sure, I might need to consult Alan once he's back for certain strings.

Reply:
yeah, let's use the same text we use in ADS for this purpose: "Add Connection".